src/cljs/reagent_toolbox_docs/components/drawer.cljs

```diff
@@ -0,0 +1,102 @@
+;;;; Copyright © 2017 Flexpoint Tech Ltd
+
+(ns reagent-toolbox-docs.components.drawer
+  (:require [reagent-toolbox-docs.layout :as layout]
+            [reagent-toolbox-docs.ui :as ui]))
+
+(defn view
+  []
+  (fn []
+    [:article
+     [:h1 "Drawer"]
+     [:p "The "
```

Comment:
I normally use a maximum width of 120 and try to have code as compact as possible. I allow longer lines if that's the best way to go or it's not important.